presto-spi/src/main/java/io/prestosql/spi/security/Identity.java

```diff
@@ -66,25 +65,6 @@ public ConnectorIdentity toConnectorIdentity(String catalog)
         return new ConnectorIdentity(user, principal, Optional.ofNullable(roles.get(catalog)));
     }
 
-    @Override
-    public boolean equals(Object o)
```

Comment:
> It is questionable if Identities with different principals and
same user are still same objects. Such case should be solved in place of
usage, not in the entity. So far these methods were not used (Identity
or any object that wraps it was not stored in any collection).

how do you verify this?

also, https://github.com/prestosql/presto/pull/113#issuecomment-459305288